lib/libzpool/Makefile.am

```diff
@@ -12,11 +17,16 @@ AM_CFLAGS += $(NO_UNUSED_BUT_SET_VARIABLE)
 # Includes kernel code generate warnings for large stack frames
 AM_CFLAGS += $(FRAME_LARGER_THAN)
 
-AM_CFLAGS += -DLIB_ZPOOL_BUILD
+AM_CFLAGS += -DLIB_ZPOOL_BUILD -Wframe-larger-than=32768
```

Comment:
this will conflict with line 13/18

Reply:
it should be 2048. this is a requirement for the zstd code. usually it was messed up to by your clean up procedure. in the early versions is had 32768, but later i changed it to 2048 which was a safe value for all architecures i tested. i can remove it, but i may raise up warnings depending on the archticture and compiler version. default seem to be 4096 now. according to the configure script. so usually its obsolete

Reply:
Well, what I meant was that you will have frame-larger-than be set twice and thus overwritten regardless of $(FRAME_LARGER_THAN)

Reply:
i have seen it. i removed this line in latest code now